crates/biome_js_formatter/src/js/expressions/call_arguments.rs

```diff
@@ -902,14 +902,16 @@ fn is_simple_ts_type(ty: &AnyTsType) -> bool {
     //     Foo<number, string> => Foo<number, string>
     let extracted_generic_type = match &extracted_array_type {
         Some(AnyTsType::TsReferenceType(generic)) => {
-            generic.type_arguments().and_then(|type_arguments| {
+            if let Some(type_arguments) = generic.type_arguments() {
```

Comment:
The issue was that this `and_then` would return `None` if the `type_arguments` was `None`, but we actually want it to return the `extracted_array_type`.

This could possibly be written a different way, but I think it's clear enough as is, and definitely avoids this implicit issue now.